libs/api/domains/hms-loans/src/lib/models/loanOverview.model.ts

```diff
@@ -0,0 +1,142 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('HmsLoansVedstadir')
+export class Vedstadir {
+  @Field(() => String, { nullable: true })
+  fastanumer?: string
+
+  @Field(() => String, { nullable: true })
+  vedstadurHeiti?: string
+
+  @Field(() => String, { nullable: true })
+  vedstadurSVfelag?: string
+
+  @Field(() => String, { nullable: true })
+  epilog?: string
+
+  @Field(() => String, { nullable: true })
+  svFelagsNumer?: string
+}
+
+@ObjectType('HmsLoansMedgreidendur')
+export class Medgreidendur {
+  @Field(() => String, { nullable: true })
+  nafnMedgreidandi?: string | null
+
+  @Field(() => String, { nullable: true })
+  medGreidandi?: string | null
+}
+
+@ObjectType('HmsLoansLoanOverview')
+export class LoanOverview {
```

Comment:
I would argue that this object should just be named `Loan`. It is implied to being an overview if you have list of `Loan` objects.

```suggestion
@ObjectType('HmsLoansLoan')
export class Loan {
```

Reply:
Fixed